src/Raven.Studio/typescript/models/database/cluster/clusterNode.ts

```diff
@@ -62,6 +63,14 @@ class clusterNode {
         });
         
         this.osIcon = ko.pureComputed(() => clusterNode.osIcon(this.osInfo().Type));
+
+        this.memoryTitle =
+            `<div class="text-left margin-left-sm">
+                 <ul class='margin-top margin-left margin-right-sm no-padding-left'>
+                     <li>The amount of memory that RavenDB will use on the node <br> for managed memory (.NET) and unmanaged memory <br> (handled by RavenDB).</li> 
+                     <li>This limit, which is set by your license, <br> is Not enforced on the mmap files managed by the OS.</li>
```

Comment:
```suggestion
                     <li>This limit, which is set by your license, <br> is not enforced on the mmap files managed by the OS.</li>
```

That is a style change

Reply:
done